src/cli.ts

```diff
@@ -21,6 +23,7 @@ const parsed = yargs(hideBin(process.argv))
   })
   .option('endpoint', {
     array: true,
```

Comment:
should this be mentioned in a readme that it is now an array?

Reply:
I think the README needs a lot of work as it doesn't even mention we can configure with envs. Another PR.